File: 프로그래머스/lv2/12945. 피보나치 수/피보나치 수.js
function solution(n) {
    var dp = Array(n+1)
    dp[0]=0
    dp[1]=1
    for(let i=2;i<dp.length;i++){
        dp[i] = (dp[i-1]+dp[i-2])%1234567
    }
    return dp[dp.length-1];
}

File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.py
def solution(n):
    answer = 0
    f = [0 for _ in range(100050)]
    f[0] = 0
    f[1] = 1
    
    for i in range(2, n+1):
        f[i] = f[i-1] + f[i-2]
        
    return f[n]%1234567